### ISSUE TITLE
[Legitimate Site Blocked] openten.io

### ISSUE THREAD

**Issue by opentendao** [MetaMask/eth-phishing-detect#11947]
https://openten.io/ Hello, we just bought the domain and were not warned about the problems. what are the options to unblock this link? 
simply.com was bought in the councils!

**Comment by AlexHerman1** [comment]
Hi there,

If you are hosting something here, please create a new issue and we will be happy to review for unblocking. Thanks.

**Comment by opentendao** [comment]
We use hosting our web solution on simply.com. When can unblock the domain?
